examples/ChatQnA/ChatQnA_Guide.rst

```diff
@@ -142,20 +132,85 @@ See :doc:`/GenAIExamples/supported_examples`
 Architecture
 ************
 
-TODO: Includes microservice level graphics.
+The ChatQnA architecture is displayed below:
+![alt text](image.png)
+
+.. figure:: /GenAIExamples/ChatQnA/assets/img/chatqna_flow_chart.png
+   :alt: ChatQnA Architecture Diagram
+
+.. code-block:: python
```

Comment:
What graph drawing language is this?  It looks a bit like graphviz, but syntax isn't correct.  If it were, you could use the graphviz directive instead of a code block:

```
.. graphviz::
```

Reply:
I'm having issues getting this to render as a mermaid diagram myself.  Let's remove the diagram for now so we can get this document merged (without that diagram).